docs/static-website.md

```diff
@@ -101,6 +101,35 @@ serverless landing:upload
 
 This command only takes seconds: it directly uploads files to S3 and clears the CloudFront cache.
 
+## Variables
+
+All static-website constructs expose the following variables:
+
+- `cname`: the domain name of the resource, such as `d111111abcdef8.cloudfront.net`
+
+This can be used to reference the bucket from Route53 configuration, for example:
+
+```yaml
+constructs:
+    landing:
+        type: static-website
+        path: public
+
+resources:
+  Resources:
+    Route53Record:
+      Type: AWS::Route53::RecordSet
+      Properties:
+        HostedZoneId: ZXXXXXXXXXXXXXXXXXXJ
+        Name: app.mydomain.
+        Type: A
+        AliasTarget:
+          HostedZoneId: ZXXXXXXXXXXXXXXXXXXJ
```

Comment:
Maybe you could hardcode the HostedZoneId. The HostedZoneId here will always be the ID of the Route53 zone for Cloudfront (cloudfront.net) which never changes. The ID of this hosted zone is : Z2FDTNDATAQYW2

It will make it easier for someone who doesn't know Cloudfront very well to create the custom domain !

Reply:
Sounds like a very good idea!